generator/testdata/google/example/library/v1/library_service_stub.gapic.cc.baseline

```diff
@@ -124,14 +126,208 @@ class DefaultLibraryServiceStub : public LibraryServiceStub {
     ::google::example::library::v1::GetBookRequest const& request,
     ::google::example::library::v1::Book* response) override {
     grpc::ClientContext grpc_ctx;
-    context->PrepareGrpcContext(&grpc_context);
+    context->PrepareGrpcContext(&grpc_ctx);
     return google::gax::GrpcStatusToGaxStatus(grpc_stub_->GetBigBook(&grpc_ctx, request, response));
   }
 
  private:
   std::unique_ptr<::google::example::library::v1::LibraryService::StubInterface> grpc_stub_;
 }; // DefaultLibraryServiceStub
 
+class RetryLibraryServiceStub : public LibraryServiceStub {
+ public:
+  RetryLibraryServiceStub(std::unique_ptr<LibraryServiceStub> stub_,
+                         gax::RetryPolicy const& retry_policy,
+                         gax::BackoffPolicy const& backoff_policy) :
+            next_stub_(std::move(stub)),
+            retry_policy_(retry_policy.clone()),
```

Comment:
s/retry_policy_/default_retry_policy_/ maybe? Ditto for `backoff_policy_`?

Reply:
Yeah, that's a more helpful naming convention.